create-quasar/utils/index.js

```diff
@@ -218,6 +220,49 @@ module.exports.ensureOutsideProject = function () {
   }
 }
 
+async function getShellResponse(command) {
+  try {
+    await execComm(command)
+    return true
+  } catch (e) { 
+    return false
+  }
+}
+
+module.exports.extendJsonFile = async function (filePath, newData) {
+  if (Object.keys(newData).length > 0) {
+    const localNpmResponse = await getShellResponse('npm list --depth=0 | grep @quasar/cli')
+    const localPnpmResponse = await getShellResponse('pnpm list --depth=0 | grep @quasar/cli')
+    const localYarnResponse = await getShellResponse('yarn list --depth=0 | grep @quasar/cli')
+    const globalPnpmResponse = await getShellResponse('pnpm -g list --depth=0 | grep @quasar/cli')
+    const globalYarnResponse = await getShellResponse('yarn global list --depth=0 | grep @quasar/cli')
+    const globalNpmResponse = await getShellResponse('npm list -g --depth=0 | grep @quasar/cli')
+    if (globalNpmResponse || globalYarnResponse || globalPnpmResponse || localYarnResponse || localPnpmResponse || localNpmResponse) {
```

Comment:
Checks to discover if `@quasar/cli` has been installed globally are the right path, but you can probably also check directly if a `quasar` bin exists on the global scope instead, which is a package manager-independent way to detect the global installation of that package

Local installation of `@quasar/cli` isn't officially supported for now, so we shouldn't support it either